src/TesApi.Web/BatchScheduler.cs

```diff
@@ -1106,7 +1106,7 @@ private async Task<CloudTask> ConvertTesTaskToBatchTaskAsync(string taskId, TesT
             sb.AppendLinuxLine($"chmod -R o+rwx $AZ_BATCH_TASK_WORKING_DIR/wd && \\");
             sb.AppendLinuxLine($"export TES_TASK_WD=$AZ_BATCH_TASK_WORKING_DIR/wd && \\");
             sb.AppendLinuxLine($"write_ts ExecutorStart && \\");
-            sb.AppendLinuxLine($"docker run --rm {volumeMountsOption} --entrypoint= --workdir / {executor.Image} {executor.Command[0]}  \"{string.Join(" && ", executor.Command.Skip(1))}\" && \\");
+            sb.AppendLinuxLine($"docker run --rm {volumeMountsOption} --entrypoint= --workdir {executor.Workdir ?? "/"} {executor.Image} {executor.Command[0]}  \"{string.Join(" && ", executor.Command.Skip(1))}\" && \\");
```

Comment:
Suggestion: use `string.IsNullOrWhitespace`

Reply:
It would change the default behavior right? running on the container workdir vs on root, are we good with that?

Reply:
I will talk with the team on Friday about this, but the TES spec is clear that if this parameter is not set then the container default is to be used.

Reply:
@BMurri Done